### DISCUSSION TITLE
2016/05/20/welcome-to-jekyll

### DISCUSSION THREAD

**Discussion by giscus** [toshimaru/jekyll-theme-classless-simple#13] [Announcements]
# 2016/05/20/welcome-to-jekyll

You’ll find this post in your _posts directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run jekyll serve, which launches a web server and auto-regenerates your site when a file is updated.

http://localhost:4000/2016/05/20/welcome-to-jekyll.html

**Comment by toshimaru** [comment]
test comment

**Reply by toshimaru** [reply]
test reply
